http2-trace-analysis/mosaic_support.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+DEFAULT_WEBSITE="about:blank"
+# Point to current directory
+export SSLKEYLOGFILE="$(dirname $0)/keylogfile.txt"
+
+case "$(uname -s)" in
+Linux*)
+	BROWSER_BINS=(chromium google-chrome firefox brave librewolf icecat)
+	# Kill all browser processes
+	for BROWSER_BIN in "${BROWSER_BINS[@]}"; do
+		if command -v "$BROWSER_BIN" &>/dev/null; then
+			killall -9 "$BROWSER_BIN" &>/dev/null
+		fi
+	done
```

Comment:
I thought about this: can we actually just terminate the script asking the user to shut down their browser in case one is open instead of killing the process (output which browser they have to terminate as well)? We would avoid confusion among the people whose browser is all of the sudden turned off this way